prompb/types.proto

```diff
@@ -56,6 +56,9 @@ message ReadHints {
   string func = 2;    // String representation of surrounding function or aggregation.
   int64 start_ms = 3; // Start time in milliseconds.
   int64 end_ms = 4;   // End time in milliseconds.
+  repeated string grouping = 5; // List of label names of any fields used in aggregation.
+  bool by = 6; // Indicate whether it is without or by, it will be false if grouping is not used.
+  int64 range_ms = 7; // Temporal aggregation time window in millisecond.
```

Comment:
This is a bit wordy, and we don't use these terms in the docs. "Range vector selector range" covers it.